spring-cloud-gcp-autoconfigure/src/main/java/org/springframework/cloud/gcp/autoconfigure/vision/CloudVisionAutoConfiguration.java

```diff
@@ -117,4 +104,32 @@ public Executor cloudVisionExecutor() {
 		ackExecutor.setThreadNamePrefix("gcp-cloud-vision-ocr-executor");
 		return ackExecutor;
 	}
+
+	/**
+	 * Configures the {@link DocumentOcrTemplate} class.
+	 */
+	@EnableConfigurationProperties(CloudVisionProperties.class)
```

Comment:
I think it's enough that this annotation exists on the parent class.

Reply:
Done.